src/storage/page_buffer.c

```diff
@@ -370,11 +370,11 @@ struct pgbuf_bcb
   int ain_tick;			/* age of AIN when this BCB was inserted into AIN list */
   int avoid_dealloc_cnt;	/* increment before obtaining latch to avoid dellocation; decrement after latch is
 				 * obtained */
-  unsigned dirty:1;		/* Is page dirty ? */
+  volatile unsigned dirty;	/* Is page dirty ? */
```

Comment:
Let's just use `bool` for the bit fields. So, it would be:
```C
volatile bool dirty;
bool avoid_victim;
bool async_flush_request;
bool victim_candidate;
```